installer/install.iss

```diff
@@ -3327,6 +3337,18 @@ begin
         InstallWindowsTerminalFragment();
     end;
 
+    {
+        Optionally "skip" installing bundled SSH binaries conflicting with external OpenSSH:
+    }
+
+#ifdef DELETE_OPENSSH_FILES
+    if RdbSSH[GS_ExternalOpenSSH].Checked then begin
```

Comment:
Should also check for `SSHChoicePage<>NIL`.

Reply:
Isn't it implied?, how could you get this Checked if there was no form to select it?

Reply:
There is third option `RdbSSH[GS_ExternalOpenSSH]` was never initialized, thus referencing to `Checked` property is not allowed. https://github.com/git-for-windows/git/issues/3368

Reply:
Sorry about that; I didn't have a Windows 7 installation (or any Windows installation prior to the Win32 SSH rollout) around to test this with and it didn't occur to me that I could've used the [IE Developer VMs](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/) to test this, until now. Interestingly, the Win7 IE 11 VM I downloaded comes with [OpenSSH preinstalled](https://www.mls-software.com/opensshd.html).